open-metadata-implementation/admin-services/admin-services-spring/src/main/java/org/odpi/openmetadata/adminservices/spring/ConfigResource.java

```diff
@@ -46,16 +46,20 @@ public OMAGServerConfigResponse getStoredConfiguration(@PathVariable String user
         return adminAPI.getStoredConfiguration(userId, serverName);
     }
     /**
-     * Return the configuration documents stored on the platform
+     * Return the server configuration documents stored on the platform
      *
      * @param userId  user that is issuing the request
      * @return OMAGServerConfigs properties or
      * OMAGNotAuthorizedException the supplied userId is not authorized to issue this command or
      * OMAGInvalidParameterException invalid parameter occurred while processing.
      */
-    @GetMapping(path = "/configurations")
+    @GetMapping(path = "/platform/server-configurations")
```

Comment:
should be somethin like `.../configurations/all`

Reply:
I changed to /configurations , I think this implies that all the configurations will be returned, we we need to filter , page, etc in the future I suggest we add query params for that.